### DISCUSSION TITLE
Containerization

### DISCUSSION THREAD

**Discussion by ncipollina** [DavidRieman/WheelMUD#131] [Ideas]
I had a few issues in getting the app running due to the. Net core version requirement for ravendb. Any thought to adding Docker support so the server can run in a container and remove that dependency?

**Comment by DavidRieman** [comment]
Could you describe what sort of trouble you ran into?  Was it a runtime error coming from RavenDB at startup?

How does Docker help here?  I'm vaguely aware of Docker but haven't used it before;  is it basically setting devs up with a guaranteed mini cloud environment, so no dependency setup?  Can devs run/test this way with no server costs?  If we can have a low impact way to support Containerized mode while continuing to support the existing app / Windows Service modes, that does seem worthwhile.  I'm interested to hear more  :)

**Comment by ncipollina** [comment]
It's been a minute since I added this discussion. I don't remember anymore what I ran into, but if RavenDB is a requirement, if you run this using something like docker compose or even a local k8's solution like minikube, you could have a RavenDB instance start with the server instance.
